rootfs/etc/haproxy/template/haproxy.tmpl

```diff
@@ -127,6 +127,9 @@ frontend httpfront
 {{ end }}
 {{ end }}
     default_backend {{ $ing.DefaultServer.RootLocation.Backend }}
+{{ if ne $cfg.HTTPLogFormat "" }}
+    log-format {{ $cfg.HTTPLogFormat }}
+{{ end }}
```

Comment:
Move all the `log-format` to within the `if syslog` condition in the same group. If there are no syslog endpoint, `log-format` has no effect.